oauth2Flows.md

```diff
@@ -0,0 +1,25 @@
+# Flow diagrams
```

Comment:
you can move this file to a folder called `flows` and we can keep it - this is good doc to have anyway

Reply:
changes made , added todo's as well